README.md

```diff
@@ -75,9 +75,12 @@ The bot will send you a private message (make sure your Discord settings allow D
 
 If you want to view command usage or see the available options, type `.au` or `.au help` in your Discord channel.
 
+## Galactus 
+
+Galactus is a system used to speed up people's muting (which is normally slow to due to Discord limiting bot actions to 5 calls in 5 seconds per server) by having the user provide a second token (created by the user) to the bot which helps mute/deafen players quicker. You can find a guide to set this up [here.](https://youtu.be/jKcEW5qpk8E)
 ## Commands
 
-The Discord Bot uses the `.au` prefix for any commands
+The Discord Bot uses the `.au` prefix for any commands my default; if you change your prefix remember to replace `.au` with your custom prefix
```

Comment:
Typo, I think "my" -> "by"

Maybe also mention that you can @mention the bot if you ever forget your prefix? Small feature I might not have written down anywhere...